docs/modules/development/pages/rest/health_rest.adoc

```diff
@@ -0,0 +1,42 @@
+
+= Health Rest Service
+
+Service that allows health rest calls
+
+Currently implemented on Minion and Sentinel by default
+
+Examples:
+
+* /rest/health
+* /rest/health/probe
+* /rest/health?tag=local
+* /rest/health/probe?tag=local
+
+== GETs (Reading Data)
+
+[options="header, %autowidth"]
+|===
+| Resource             | Description
+| `/health`            | Health check results for components. Include health check result details for each element.
+| `/health/prob`       | Health check probe for components. Only indicate if the health is good or not. 
+|===
+
+The following table shows all supported query string parameters and their default values.
+
+[options="header"]
```

Comment:
Please update table formatting style as described in the "Develop Documentation" section of the docs (https://docs.opennms.com/horizon/29.0.1/development/docs/writing.html#gd-docs-tables). We have recently updated how we do tables.

Reply:
Updated base on the documentation